main.go

```diff
@@ -0,0 +1,230 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slackevents"
+	"github.com/slack-go/slack/socketmode"
+)
+
+func main() {
+	// Load env
+	godotenv.Load(".env")
+	token := os.Getenv("SLACK_AUTH_TOKEN")
+	appToken := os.Getenv("SLACK_APP_TOKEN")
+
+	// Set debug to true for dev
+	client := slack.New(token, slack.OptionDebug(true), slack.OptionAppLevelToken(appToken))
```

Comment:
Question: would we need to transfer between `dev` mode and `production` mode here ?

Reply:
Add a new ENV `APP_ENV` for environment indication